data/libs/SpaceStation.lua

```diff
@@ -697,6 +697,17 @@ Event.Register("onLeaveSystem", function (ship)
 	if ship ~= Game.player then return end
 	destroySystem()
 end)
+
+Event.Register("onShipDestroyed", function (ship, _)
+	for _,local_police in pairs(police) do
+		for k,police_ship in pairs(local_police) do
+			if (v == police_ship) then
```

Comment:
Oooh, right :-)

On Wed, May 25, 2016 at 2:06 PM Karl F <notifications@github.com> wrote:

> In data/libs/SpaceStation.lua
> <https://github.com/pioneerspacesim/pioneer/pull/3709#discussion_r64560074>
> :
>
> > @@ -697,6 +697,17 @@ Event.Register("onLeaveSystem", function (ship)
> >  	if ship ~= Game.player then return end
> >  	destroySystem()
> >  end)
> > +
> > +Event.Register("onShipDestroyed", function (ship, _)
> > +	for _,local_police in pairs(police) do
> > +		for k,police_ship in pairs(local_police) do
> > +			if (v == police_ship) then
>
> Maybe I'm not thinking straight, but v isn't defined, right? It should be
> ship instead?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/pioneerspacesim/pioneer/pull/3709/files/db8bdd5e4e1237b12e483d5da356444055ce6f9e#r64560074>
>